pygac/reader.py

```diff
@@ -820,7 +820,26 @@ def get_angles(self):
             arr[self.mask] = np.nan
 
         return sat_azi, sat_zenith, sun_azi, sun_zenith, rel_azi
+  
+    def _get_true_median(self, input_array, q=50, interpolation='nearest'):
```

Comment:
Oh wait :D If you have some spare time left...

This method doesn't access any class attribute, so I'd propose to move it out of the class to the module level. Or maybe to `pygac.utils`?